test/boost/expr_test_utils.hh

```diff
@@ -68,6 +68,115 @@ inline constant make_text_const(const sstring_view& text) {
     return constant(make_text_raw(text), utf8_type);
 }
 
+struct evaluation_inputs_data {
+    std::vector<bytes> partition_key;
+    std::vector<bytes> clustering_key;
+    std::vector<managed_bytes_opt> static_and_regular_columns;
+    ::shared_ptr<selection::selection> selection;
+    query_options options;
+};
+using column_values = std::map<sstring, raw_value>;
+
+// Creates evaluation_inputs that can be used to evaluate columns and bind variables using evaluate()
+inline std::pair<evaluation_inputs, std::unique_ptr<evaluation_inputs_data>> make_evaluation_inputs(
```

Comment:
Such a large function shouldn't be inline, it increases compilation time.

Reply:
At first I tried to add a matching .cc file and link it to the test, but I couldn't find an easy way to do it, so I decided that this is good enough.
Other header files in test/boost also have all functions `inline`, so I guess that's the convention there.

Reply:
There are plenty of .cc files in test/lib. Follow how one of them is referenced in configure.py.

Reply:
`test/lib` worked :)
done.